clients/fides-js/src/fides-tcf.ts

```diff
@@ -125,41 +125,39 @@ const updateCookieAndExperience = async ({
   cookie,
   experience,
   debug = false,
-  hasValidFidesStringOverride,
   isExperienceClientSideFetched,
 }: {
   cookie: FidesCookie;
   experience: PrivacyExperience;
   debug?: boolean;
-  hasValidFidesStringOverride: boolean;
   isExperienceClientSideFetched: boolean;
 }): Promise<{
   cookie: FidesCookie;
   experience: Partial<PrivacyExperience>;
 }> => {
-  // If string override exists and is valid, the cookie has already been overridden
-  if (hasValidFidesStringOverride) {
-    // However, if we didn't get the experience until the client side fetch, we need to update the fetched
-    // experience based on the cookie here.
-    if (isExperienceClientSideFetched) {
-      debugLog(
-        debug,
-        "Overriding preferences from client-side fetched experience with cookie fides_string consent",
-        cookie.fides_string
-      );
-      const tcfEntities = buildTcfEntitiesFromCookie(experience, cookie);
-      return { cookie, experience: tcfEntities };
-    }
+  if (!isExperienceClientSideFetched) {
     // If it's not client side fetched, we don't update anything since the cookie has already
     // been updated earlier.
     return { cookie, experience };
   }
 
+  // If cookie.fides_string exists, update the fetched experience based on the cookie here.
+  if (cookie.fides_string) {
+    debugLog(
+      debug,
+      "Overriding preferences from client-side fetched experience with cookie fides_string consent",
+      cookie.fides_string
+    );
+    const tcfEntities = buildTcfEntitiesFromCookie(experience, cookie);
+    return { cookie, experience: tcfEntities };
+  }
+
   // If user has no prefs saved, we don't need to override the prefs on the cookie
   if (!hasSavedTcfPreferences(experience)) {
-    return { cookie: { ...cookie, fides_string: "" }, experience };
+    return { cookie, experience };
```

Comment:
Even though this line shouldn't be hit if we have a `fides_string`, defensively, we don't want to overwrite the `fides_string` based on prefs from experience

Reply:
hmm would this mess with our initialization logic in the case of no fides string and no saved preferences yet? I guess there'd be no fides_string variable, which might be the same as having an empty string...